src/sections/collection/CollectionInfo.tsx

```diff
@@ -14,6 +16,9 @@ export function CollectionInfo({ collection }: CollectionInfoProps) {
         {collection.affiliation && (
           <span className={styles.subtext}>({collection.affiliation})</span>
         )}
+        {!collection.isReleased && (
+          <Badge variant={DatasetLabelSemanticMeaning.WARNING}>Unpublished</Badge>
+        )}
```

Comment:
We could wrap both affiliation and badge in a flex container with a gap (or use Stack component from react-bootstrap but I think we dont have it in the Design System yet), so that we dont rely on inline displaying from span elements and Badge component.
```
<div className={styles.someContainerName}> or <Stack direction="horizontal" gap={3}>
    {collection.affiliation && (
      <span className={styles.subtext}>({collection.affiliation})</span>
    )}
    {!collection.isReleased && (
      <Badge variant={DatasetLabelSemanticMeaning.WARNING}>Unpublished</Badge>
    )}
</div>
```

Reply:
good idea, thanks 😊